# This message is used to notify the system about one or more parameter changes

uint32 dummy       # unused
